### PR TITLE
More npm dependency and nav cleanup

### PR DESCRIPTION
**Pull request by csillag**
This changes fixes all the bugs I have been seeing.

### CODE REVIEW

File: examples/package.json [hunk 1]
```diff
@@ -28,8 +28,7 @@
     "parcel": "1.12.4",
     "parcel-plugin-less-js-enabled": "1.0.2",
     "prettier": "2.1.1",
-    "react-code-prettify": "0.0.1",
     "tslint": "6.1.3",
-    "typescript": "4.0.2"
+    "typescript": "3.8.3"
```
Comment (scharf) [inline examples/package.json:32]: Is there a problem with TS 4?

**Review by scharf — COMMENTED** [review]
Where was the malformed import?

File: examples/package.json (hunk 1 above)
Comment (csillag) [reply examples/package.json:32]: No problem as such, but the latest Meteor version uses TS 3.8.3.
It's easier to keep things in sync if we use the same version everywhere.

(I really wouldn't like to run into situations where our code is working fine in half of the situations, but produces inexplicable errors in the other half of the situations...)

File: examples/src/MenuStructure.tsx [hunk 2]
```diff
@@ -8,7 +8,6 @@ import { ApplicationAnt } from './app/ApplicationAnt';
 import { LoginFormAnt } from './form/LoginFormAnt';
 import { MemTableAnt } from './memtable/MemTableAnt';
 import { MoreMenusAnt } from './menus/MoreMenusAnt';
-import { NavRefRedirect } from '../../packages/antd/src/not-antd';
```
Comment (csillag) [inline examples/src/MenuStructure.tsx:11]: This was the malformed import @scharf

**Review by scharf — APPROVED** [review]
LGTM

**Pull request merged**